docs/v4.0-draft1/resources-and-data-models/aisp/standing-orders.md

```diff
@@ -104,17 +104,16 @@ If the ReadPAN permission is granted by the PSU, the ASPSP may choose to populat
+| Classification | 0..1 |OBReadStandingOrder6/Data/MandateRelatedInformation/Classification|Specifies the type of direct debit amount, such as fixed or variable. |OBClassification1Code |FIXE<br>USGB<br>VARI| |
+| CategoryPurposeCode | 0..1 |OBReadStandingOrder6/Data/MandateRelatedInformation/CategoryPurposeCode |Specifies the category purpose. For more information see `ExternalCategoryPurpose1Code` [here](https://github.com/OpenBankingUK/External_Interal_CodeSets) |OBCategoryPurpose1Code | | |
 | FirstPaymentDate | 0..1 |OBReadStandingOrder6/Data/MandateRelatedInformation/FirstPaymentDate |The date on which the first payment for a Standing Order schedule will be made. |ISODate | | |
 | FinalPaymentDate | 0..1 |OBReadStandingOrder6/Data/MandateRelatedInformation/FinalPaymentDate |The date on which the final payment for a Standing Order schedule will be made. |ISODate | | |
-| Frequency | 0..1 |OBReadStandingOrder6/Data/MandateRelatedInformation/Frequency |A code indicating the frequency of payment for the Standing Order. Values:<br>ADHO<br>YEAR<br>DALI<br>INDA<br>MNTH<br>QURT<br>MIAN<br>WEEK |OBFrequencyPeriodType | | |
-| PeriodType | 1..1 |OBReadStandingOrder6/Data/MandateRelatedInformation/PeriodType |A code indicating the period type for the Standing Order. Values:<br>ADHO<br>YEAR<br>DALI<br>INDA<br>MNTH<br>QURT<br>MIAN<br>WEEK |OBFrequencyPeriodType | | |
-| CountPerPeriod | 1..1 |OBReadStandingOrder6/Data/MandateRelatedInformation/CountPerPeriod | |int32 | |
-| PointInTimeType | 1..1 |OBReadStandingOrder6/Data/MandateRelatedInformation/PointInTimeType |A code indicating the point in time for payment of the Standing Order. Values:<br>ADHO<br>YEAR<br>DALI<br>INDA<br>MNTH<br>QURT<br>MIAN<br>WEEK |OBFrequencyPeriodType | | |
-| PointInTime | 1..1 |OBReadStandingOrder6/Data/MandateRelatedInformation/PointInTime |ISODateTime | | |
-| Reason| 0..1 |OBReadStandingOrder6/Data/MandateRelatedInformation/Reason | |Max256Text | |
+| Frequency | 0..1 |OBReadStandingOrder6/Data/MandateRelatedInformation/Frequency |A code indicating the frequency of payment for the Standing Order. |OBFrequency6 | | |
+| PeriodType | 1..1 |OBReadStandingOrder6/Data/MandateRelatedInformation/Frequency/PeriodType |A code indicating the period type for the Standing Order. |OBFrequency6Code |ADHO<br>YEAR<br>DALI<br>INDA<br>FRTN<br>MNTH<br>QURT<br>MIAN<br>WEEK  | |
+| CountPerPeriod | 1..1 |OBReadStandingOrder6/Data/MandateRelatedInformation/Frequency/CountPerPeriod | |Int32 | |
+| PointInTime | 1..1 |OBReadStandingOrder6/Data/MandateRelatedInformation/Frequency/PointInTime |Further information on the exact point in time the event should take place |Exact2NumericText | |
+| Reason| 0..1 |OBReadStandingOrder6/Data/MandateRelatedInformation/Reason |Reason for the direct debit mandate to allow the user to distinguish between different mandates for the same creditor. |Max256Text | |
```

Comment:
remove mention of direct debit from standing order data table (in 'Reason' description)